script/old/refresh_controllers

```diff
@@ -186,7 +186,7 @@ end
 
 norobots = [
   "/ajax/",
-  "/api/"
+  "/api2/"
```

Comment:
@pellaea Are the scripts in `script/old` used?
If yes, what's intended here (`api` vs `api2`)?